httpservice/src/usHttpServletRequest.cpp

```diff
@@ -258,7 +262,7 @@ long long HttpServletRequest::GetDateHeader(const std::string& name) const
                                            &second) == 6)) {
 
     month = -1;
-    for (std::size_t i = 0; i < num_months; ++i)
+    for (int i = 0; i < num_months; ++i)
```

Comment:
Thanks for fixing that. However, now gcc complains in the statement below which assigns `i` to `month`. I suggest to keep `std::size_t` in the for loop and use
```
month = static_cast<int>(i);
```
before breaking the loop. Thanks!